Engine/ScriptAnalyzerEngine.csproj

```diff
@@ -70,7 +71,7 @@
     <Compile Include="Loggers\WriteObjectsLogger.cs" />
     <Compile Include="ScriptAnalyzer.cs" />
     <Compile Include="SpecialVars.cs" />
-    <Compile Include="Strings.cs">
+    <Compile Include="Strings.Designer.cs">
```

Comment:
This line was actually generated after I changed something in Strings.resx and built the project.